src/libopensc/card-cardos.c

```diff
@@ -159,49 +190,110 @@ static int cardos_have_2048bit_package(sc_card_t *card)
 	return 0;
 }
 
+
+/* Called from cardos_init for old cards, from cardos_cardctl_parsed_token_info for new cards */
+/* TODO see if works from old cards too */
+static int cardos_add_algs(sc_card_t *card, unsigned long flags, unsigned long ec_flags, unsigned long ext_flags)
+{
+
+	cardos_data_t * priv = (cardos_data_t *)card->drv_data;
```

Comment:
clang-format suggested the following change:


```
-	cardos_data_t * priv = (cardos_data_t *)card->drv_data;
+	cardos_data_t *priv = (cardos_data_t *)card->drv_data;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->